worldedit-forge/src/main/resources/mcmod.info

```diff
@@ -14,7 +14,8 @@
         "Forge@[${forgeVersion},)"
     ],
     "dependencies": [
-        "Forge@[${forgeVersion},)"
+        "Forge@[${forgeVersion},)",
+        "ForgeMultipart"
```

Comment:
is FMP not part of forge? what happens when there's a forge server without FMP then, does WE not load due to a missing dependency?